src/gridfs/file.rs

```diff
@@ -618,7 +622,7 @@ impl GfsFile {
             "chunkSize" => (self.chunk_size),
             "length" => (self.len),
             "md5" => (self.md5.to_owned()),
-            "uploadDate" => (self.upload_date.as_ref().unwrap().clone())
+            "uploadDate" => (self.upload_date.unwrap())
```

Comment:
Clippy noticed that the date type implements `Copy`, so rather than taking a reference and cloning it, I updated it to implicitly copy.